securedrop/passphrases.py

```diff
@@ -0,0 +1,123 @@
+from pathlib import Path
+from random import SystemRandom
+from typing import Optional, NewType, Set
+
+from typing import Dict
+from typing import List
+
+from sdconfig import config
+
+# A list of words to be used by as a passphrase
+# For example: "recede anytime acorn durably discuss"
+# More details at https://www.rempe.us/diceware/#eff
+DicewarePassphrase = NewType("DicewarePassphrase", str)
+
+
+_current_generator = None  # type: Optional["PassphrasesGenerator"]
+
+
+class InvalidWordsListError(Exception):
+    pass
+
+
+class PassphrasesGenerator:
```

Comment:
This is a nit, but `InvalidWordsListError` and `PassphrasesGenerator` sound odd to me. I'd expect `InvalidWordListError` and `PassphraseGenerator`.

Reply:
Yeah I can change that